src/api/DataHandler.ts

```diff
@@ -1,11 +1,61 @@
 import fetch from 'node-fetch';
 
-const IP = "http://localhost:8081" //"80.167.223.178";
+const api = "localhost:3306" //"80.167.223.178";
 
 enum HttpRequestType {
     Get,
     Post
 }
+//@ts-ignore
+function Login(email: string, password: string){
+    return new Promise((resolve, rejects) => {
```

Comment:
Defauly syntax is "reject", although I'm fine as long as it's consistant with either reject, or rejects.

Reply:
stuck with rejects